components/pom.xml

```diff
@@ -152,7 +152,7 @@
             <plugin>
                 <groupId>org.apache.camel</groupId>
                 <artifactId>camel-bundle-plugin</artifactId>
-                <version>${camel-version}</version>
+                <version>3.22.2</version>
                 <extensions>false</extensions>
```

Comment:
We need to use the maven-bundle-plugin instead https://github.com/apache/camel-karaf/blob/main/pom.xml#L702-L707